### PR TITLE
[4.11] Add disconnected STS jobs (with endpoints)

### PR DESCRIPTION
**Pull request by yunjiang29**

### CODE REVIEW

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml [hunk 1]
```diff
@@ -150,6 +150,32 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-byo-route53
+- as: aws-ipi-disconnected-cco-manual-sts-w-ep-p2-f7
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml:153]: How about rename it to `aws-ipi-disconnected-cco-manual-sts-endpoint-p2-f7`

Comment (yunjiang29) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml:153]: `aws-ipi-disconnected-cco-manual-sts-endpoint-p3-f28-destructive` exceeds the maximum length limitation.
How about `aws-ipi-disconnected-cco-manual-sts-ep-p3-f28-destructive`?

Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml:153]: ah, forget the `destructive` one. Yeah, `aws-ipi-disconnected-cco-manual-sts-ep-p3-f28-destructive` sounds good to me.

Comment (yunjiang29) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml]: Updated.

**Pull request merged**